notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
<p>There is no need to duplicate the output in the markdown cell, it will be shown in the rendered notebook. Please remove the markdown copy. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>